utils/chat-utils.js

```diff
@@ -4,10 +4,15 @@ function isAskingLeni(message) {
 	);
 }
 
-// TODO: Once we have classrooms id we can put them in the .env file and use it here
 // eslint-disable-next-line no-unused-vars
 function isAskingInAClassroom(message) {
-	return true;
+	return process.env.CLASSROOM_IDS.includes(
+		message?.channelId,
+	);
+	// if (!process.env.CLASSROOM_IDS) {
+	// 	return false;
+	// }
+	// return process.env.CLASSROOM_IDS.some(id => id === message.channelID);
```

Comment:
```suggestion
```

Reply:
Let's delete this code, it's unnecesary and you includes solution is far better 😅